include/dbclean.php

```diff
@@ -18,16 +23,20 @@ function dbclean_run(&$argv, &$argc) {
 		unset($db_host, $db_user, $db_pass, $db_data);
 	}
 
-	load_config('config');
-	load_config('system');
```

Comment:
So load_config() is deprecated? Can it this then please be marked with `@deprecated` ?